hooks/favorites.js

```diff
@@ -0,0 +1,60 @@
+import { useState, useEffect } from "react";
```

Comment:
not bad but this is pretty much a copy of everything I wrote which is why you have a bunch of code that doesn't quite apply to favorite list. can you identify those lines?

Reply:
Actually i didn't understand how to implement those things so copied your cart item theme and tried to work on this . And i removed lots of cart item codes but as you said there are still available i'll check those codes and try to solve .. 
Thanks